indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -328,7 +525,7 @@ class BookingEdit extends React.Component {
                             </Singular>
                             <Plural>
                                 <Param name="count" value={this.numberOfConflicts} /> occurrences of your
-                                new booking might conflict with existing bookings.
+                                new booking might conflict with existing bookings/unbookable periods.
```

Comment:
What about just saying "...are unavailable due to conflicts"? unbookable periods are not very common (probably many people never heard of them) and mentioning them here might confuse people.